appyters/Gene_Age_Trends_Appyter/Age_Gene_Appyter.ipynb

```diff
@@ -0,0 +1,1772 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from appyter import magic \n",
+    "magic.init(lambda _= globals: _())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from sklearn.preprocessing import QuantileTransformer\n",
+    "from sklearn.decomposition import PCA\n",
+    "from sklearn.linear_model import LinearRegression\n",
+    "from sklearn.metrics import r2_score\n",
+    "from sklearn.manifold import TSNE\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "import pandas as pd\n",
+    "import umap\n",
+    "from pycombat import Combat\n",
+    "from scipy.optimize import curve_fit\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%appyter hide_code\n",
+    "{% do SectionField(name='section0', \n",
+    "                   title='Select Species', \n",
+    "                   subtitle='Select the Species that you would like to view') %}\n",
+    "{% do SectionField(name='section1', \n",
+    "                   title='Select Gene', \n",
+    "                   subtitle='Select the Gene that you would like to create a plot for') %}\n",
+    "{% do SectionField(name='section2',\n",
+    "                  title='Model Selection',\n",
+    "                   subtitle='Customize which parts of the model you would like to view'\n",
+    "                  ) %}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 38,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/markdown": [
+       "```python\n",
+       "SPECIES_CHOICE = 1\n",
+       "GENE_CHOICE = 1553\n",
+       "SPLINE_DEGREE = 2\n",
+       "```"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Markdown object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%%appyter code_exec\n",
+    "\n",
+    "SPECIES_CHOICE = {{ ChoiceField(name = 'SPECIES_CHOICE', label='Species Choice', choices={'Mouse':'1', 'Human':'2'}, default = \"Mouse\", section='section0')}}\n",
+    "GENE_CHOICE = {{ ChoiceField(name='GENE_CHOICE', label = 'Gene Choice', choices={'0610007P14Rik': '0',\n",
```

Comment:
This is a little excessive.. I think it really should be an autocompletion field instead of a choice field. Since we don't have such a field by default it would be good to try making such a field. Nonetheless this is something you may be able to address in a future PR.

Reply:
Yeah I also thought it might be a bit much given the number of genes. I'll look into creating the field.